### ISSUE TITLE
Pylance stuck on "Python extension loading"

### ISSUE THREAD

**Issue by ad12** [microsoft/pylance-release#1674]
Issue Type: <b>Bug</b>

Pylance is stuck on the "extension loading" step

Extension version: 2021.8.1
VS Code version: Code 1.59.0 (379476f0e13988d90fab105c5c19e7abc8b1dea8, 2021-08-04T23:14:40.191Z)
OS version: Darwin x64 20.4.0
Restricted Mode: No
Remote OS version: Linux x64 5.8.0-59-generic
Fetching remote diagnostics for 'SSH: torino' failed: Canceled
Remote OS version: Linux x64 5.8.0-63-generic
Fetching remote diagnostics for 'SSH: stelvio' failed: Canceled
Remote OS version: Linux x64 5.4.104+
Remote OS version: Linux x64 5.4.104+

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz (12 x 2900)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|7, 7, 6|
|Memory (System)|32.00GB (0.69GB free)|
|Process Argv|--crash-reporter-id 1572910a-3d68-41d5-8b2d-a8224b00f95b|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: torino|
|OS|Linux x64 5.8.0-59-generic|
|CPUs|Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz (32 x 1862)|
|Memory (System)|125.80GB (5.85GB free)|
|VM|0%|

Fetching remote diagnostics for 'SSH: torino' failed: Canceled

|Item|Value|
|---|---|
|Remote|SSH: stelvio|
|OS|Linux x64 5.8.0-63-generic|
|CPUs|Intel(R) Core(TM) i9-10900X CPU @ 3.70GHz (20 x 4270)|
|Memory (System)|125.50GB (30.58GB free)|
|VM|0%|

Fetching remote diagnostics for 'SSH: stelvio' failed: Canceled

|Item|Value|
|---|---|
|Remote|io-pod in io-pod (gcr.io/hai-gcp-mri/mtr-img)|
|OS|Linux x64 5.4.104+|
|CPUs|Intel(R) Xeon(R) CPU (60 x 3100)|
|Memory (System)|236.15GB (229.27GB free)|
|VM|0%|

|Item|Value|
|---|---|
|Remote|v100-pod-v3 in v100-pod (pytorch/pytorch:1.8.1-cuda11.1-cudnn8-devel)|
|OS|Linux x64 5.4.104+|
|CPUs|Intel(R) Xeon(R) CPU @ 2.30GHz (8 x 2299)|
|Memory (System)|29.40GB (2.30GB free)|
|VM|0%|
</details>

**Comment by jakebailey** [comment]
Can you provide a screenshot showing where you're seeing this?

If VS Code is saying "Python extension loading", then that sounds like an issue with the Python extension, not Pylance.

**Comment by ad12** [comment]
Yup! Adding a screenshot below - this is happening when I'm opening a Jupyter notebook, so it may be a Jupyter extension issue as well. Ive filed it [here](https://github.com/microsoft/vscode-jupyter/issues/7145)


![image](https://user-images.githubusercontent.com/12801331/129634623-d25fa097-1fa5-4a3b-b965-c2b7b253e0c4.png)

**Comment by jakebailey** [comment]
I don't think this is related to Pylance; we don't load until the Python extension loads. I will transfer this to them to see if they have any insight.

You may want to see if the Python output window provides some more info, or potentially the debug console.
